src/mgr/MgrStandby.cc

```diff
@@ -147,21 +147,21 @@ int MgrStandby::init()
 void MgrStandby::send_beacon()
 {
   assert(lock.is_locked_by_me());
-  dout(1) << state_str() << dendl;
+  dout(4) << "state: " << state_str() << dendl;
 
   set<string> modules;
   PyModules::list_modules(&modules);
   bool available = active_mgr != nullptr && active_mgr->is_initialized();
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
-	   << " modules " << modules << dendl;
+           << " modules " << modules << dendl;
```

Comment:
https://github.com/ceph/ceph/blob/master/CodingStyle#L76, and please do not send formatting-only patches unless you happen to touch that area.